--- origsrc/epplets-0.12/configure.ac	2009-01-03 03:10:06.000000000 -0600
+++ src/epplets-0.12/configure.ac	2009-01-11 16:22:14.187125000 -0600
@@ -74,7 +74,7 @@ if test "x$enable_glx" = "xyes"; then
 fi
 if test "x$enable_glx" = "xyes"; then
   GLX_LIBS="-lGL"
-  GLX_PROGS="E-OpenGL-Demo.epplet"
+  GLX_PROGS="E-OpenGL-Demo.epplet\$(EXEEXT)"
   AC_DEFINE(HAVE_GLX, 1, [GLX support])
 else
   AC_MSG_WARN([*** GL epplets will not be built ***])
@@ -146,7 +146,7 @@ fi
 # Build test programs?
 AC_ARG_WITH(test-programs,
   [  --with-test-programs    builds the test programs @<:@default=no@:>@],
-  TEST_PROGS="EppletTest.epplet EppletConfigTest.epplet", TEST_PROGS="")
+  TEST_PROGS="EppletTest.epplet\$(EXEEXT) EppletConfigTest.epplet\$(EXEEXT)", TEST_PROGS="")
 AC_SUBST(TEST_PROGS)
 
 # Should we install E-Mixer?
@@ -157,7 +157,7 @@ AC_CHECK_HEADERS(linux/soundcard.h,[P_MI
      AC_CHECK_HEADERS(dmedia/audio.h, [P_MIXER=mixer],[
       AC_CHECK_HEADERS(sys/soundcard.h, [P_MIXER=mixer]) ]) ]) ])
 if test "$P_MIXER" = "mixer"; then
-  MIXER_PROGS="E-Mixer.epplet Emix.epplet"
+  MIXER_PROGS="E-Mixer.epplet\$(EXEEXT) Emix.epplet\$(EXEEXT)"
 else
   AC_MSG_WARN([*** Mixer epplets will not be built ***])
   MIXER_PROGS=""
@@ -181,7 +181,7 @@ AC_ARG_ENABLE(cdaudio,
   [  --enable-cdaudio        build epplets using libcdaudio @<:@default=yes@:>@],,
   enable_cdaudio=yes)
 if test "x$enable_cdaudio" = "xyes"; then
-  AC_CHECK_LIB(cdaudio, cd_advance, PLAYCD_PROGS=E-PlayCD.epplet, PLAYCD_PROGS="")
+  AC_CHECK_LIB(cdaudio, cd_advance, PLAYCD_PROGS="E-PlayCD.epplet\$(EXEEXT)", PLAYCD_PROGS="")
 fi
 AC_SUBST(PLAYCD_PROGS)
 
@@ -189,7 +189,7 @@ AC_ARG_ENABLE(esd,
   [  --enable-esd            build epplets using libesd @<:@default=yes@:>@],,
   enable_esd=yes)
 if test "x$enable_esd" = "xyes"; then
-  AC_CHECK_LIB(esd, esd_open_sound, ESD_PROGS=E-SD.epplet, ESD_PROGS="")
+  AC_CHECK_LIB(esd, esd_open_sound, ESD_PROGS="E-SD.epplet\$(EXEEXT)", ESD_PROGS="")
 fi
 AC_SUBST(ESD_PROGS)
 
